example.js

```diff
@@ -71,6 +73,9 @@ const components = {
       ? BatchRoutingPanel
       : DefaultMainPanel,
   MapWindows: isCallTakerModuleEnabled ? CallTakerWindows : null,
+  MobileResultsScreen: isBatchRoutingEnabled
```

Comment:
Add this component as a required component in the comment about the creation of the `components` variable.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/e802be61940f6d6454d6c6291b61e310bdf9ae21.